gui/src/components/Taipy/Slider.tsx

```diff
@@ -245,6 +282,8 @@ const Slider = (props: SliderProps) => {
                     marks={marks}
                     valueLabelFormat={getLabel}
                     orientation={horizontalOrientation ? undefined : "vertical"}
+                    getAriaLabel={Array.isArray(parsedDefaultValue) ? ()=>"" : undefined}
```

Comment:
define the function as a const outside the component ?

Reply:
But yes.